src/Apps/Collect/Components/Collection/Header/index.tsx

```diff
@@ -345,10 +352,10 @@ export const Overlay = styled.div`
   height: 100%;
   top: 0;
   left: 0;
-  background: linear-gradient(
-    180deg,
-    rgba(0, 0, 0, 0.15) 0%,
-    rgba(0, 0, 0, 0.25) 95%
+  background-image: linear-gradient(
+    to bottom,
+    rgba(0, 0, 0, 0),
+    rgba(0, 0, 0, 0.25)
```

Comment:
Why did these properties change?

Reply:
Good q, that was part of the spec in https://app.zeplin.io/project/5aabc5e0786bbb29b6e3dc7f/screen/5cd98bcd27264913e9451971, as a note under the XL mockup

Reply:
Got it!